tests/end2end/data/prompt/clipboard.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<!-- largely copied from https://googlechrome.github.io/samples/async-clipboard/ -->
```

Comment:
Might need a license header to make distributions and such happy? Seems to be using [the Apache License](https://github.com/GoogleChrome/samples/blob/gh-pages/LICENSE) which means we'd at least need to have a `SPDX-License-Identifier: Apache-2.0` comment, and I suppose `SPDX-FileCopyrightText: Copyright 2014 Google Inc`.

I also wonder if we should trim it down a bit (e.g. is the CSS needed?), but no big deal either way.